### DISCUSSION TITLE
Search item. Ограничение на длину строки запроса

### DISCUSSION THREAD

**Discussion by efremych** [Chimildic/goofy#203] [Помощь с ошибкой]
Есть ощущение, что есть лимит на длину строки запроса на поиск (трека, например).
Вот [трек](https://open.spotify.com/track/3iEmgajStVBooooNaltdOv?si=7d2e8f9d2db94dd8) для примера.
Запрос такой: **_the retaliators the retaliators theme 21 bullets feat. mötley crüe asking alexandria ice nine kills from ashes to new_**
118 символов.
Свыше **101** символа - уже поиск не дает результат.

Вот [тут ](https://community.spotify.com/t5/Spotify-for-Developers/Is-there-a-String-Length-Limit-for-the-Search-Endpoint-Query/td-p/5400531)тоже товарищ задавался этим вопросом.

**Comment by Chimildic** [comment]
То есть вручную проверили что при 101 есть результат, а свыше ошибка 404?

**Reply by efremych** [reply]
![image](https://user-images.githubusercontent.com/76212904/187044324-ee0c4518-a5ff-4af0-b5dc-f837315fcbf9.png)
![image](https://user-images.githubusercontent.com/76212904/187044340-42fc0c13-1b7e-409b-b128-710a220ebe48.png)

Странно, десктопное приложение выдает результат при 101 символе. А запрос выше только при 100.

что-то так плохо и не охотно думается, куда и как лучше добавить обрезку эту?...
